lib/resolver/live-resolver-query.js

```diff
@@ -1,4 +1,9 @@
 export default function ({ type, target }) {
+  if (type == 'npm') {
+    // If the target looks like 'foo/bar.js', pretend it is 'foo' instead. See
+    // https://github.com/OctoLinker/browser-extension/issues/93
+    target = target.split('/')[0];
+  }
```

Comment:
Mhhhh I'm not sure if this is the right place to do this. Maybe a `shorthand-resolver` ... let me think about it  ...